vue.config.js

```diff
@@ -6,20 +6,18 @@ const defineGlobals = (config) => {
     args[0] = globals;
     return args;
   });
+
   config.module
-    .rule('babel')
-    .test(/base-scraper\.js$/)
-    .use('babel')
-    .loader('babel-loader')
-    .options({
-      presets: [['@babel/preset-env', { modules: false }]],
-      plugins: ['@babel/plugin-proposal-class-properties']
-    });
+    .rule('binary')
+    .test(/\.node$/)
+    .use()
+    .loader('node-loader');
 };
 
 module.exports = {
   pluginOptions: {
     electronBuilder: {
+      nodeIntegration: true,
```

Comment:
Just note, this is what allows us to call `fs`, for example, from the renderer process.

We talked about it once, we may one day want to invest in canceling it.